components/description-list/test/description-list.test.js

```diff
@@ -0,0 +1,42 @@
+import '../description-list-wrapper.js';
+
+import { expect, fixture, html } from '@open-wc/testing';
+import { runConstructor } from '../../../tools/constructor-test-helper.js';
+
+describe('d2l-dl-wrapper', () => {
+
+	describe('constructor', () => {
+		it('should construct', () => {
+			runConstructor('d2l-dl-wrapper');
+		});
+	});
+
+	describe('layout breakpoint', () => {
+		it('has a default value of 200', async() => {
+			const elem = await fixture(html`
+				<d2l-dl-wrapper>
+					<dl>
+						<dt>Title</dt>
+						<dd>Details</dd>
+					</dl>
+				</d2l-dl-wrapper>
+			`);
+
+			expect(elem.breakpoint).to.equal(200);
+		});
+
+		it('can be overridden by consumers', async() => {
+			const elem = await fixture(html`
+				<d2l-dl-wrapper breakpoint="300">
+					<dl>
+						<dt>Title</dt>
+						<dd>Details</dd>
+					</dl>
+				</d2l-dl-wrapper>
+			`);
+
+			expect(elem.breakpoint).to.equal(300);
+		});
+	});
```

Comment:
I wonder if it would be useful to test the `.stacked` class in these tests - e.g. manipulate the component's width, and check that the `.stacked` class is being added/removed. Then again, the visual diffs do already test that behaviour.

Reply:
James and I had chatted about that and just opted to rely on the visual diff tests.

Reply:
Yeah, the visual diff tests should cover default/consumer overridden breakpoint behavior, so unit tests checking the same thing would probably be redundant.